### PR TITLE
feat: Rename dockerfiles, use alpine-nginx

### PR DESCRIPTION
**Pull request by alexdor**

### CODE REVIEW

File: grnet/website.dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+# Set the base image to Debian
+FROM nginx:alpine
+
+################## BEGIN INSTALLATION ######################
+RUN apk add --update --no-cache ca-certificates
+
+# Forward request and error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/nginx/access.log
```
Comment (AEtherC0r3) [inline grnet/website.dockerfile]: This line is redundant

File: grnet/website.dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+RUN ln -sf /dev/stderr /var/log/nginx/error.log
```
Comment (AEtherC0r3) [inline grnet/website.dockerfile]: this one too

File: grnet/website.dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+COPY website-nginx.conf /etc/nginx/nginx.conf
+
+WORKDIR /usr/share/nginx/html
+RUN rm *
+COPY --from=asset_builder /new_website/* ./
+
+################## INSTALLATION END ######################
+
+EXPOSE 80 443
+CMD nginx -g "daemon off;"
```
Comment (AEtherC0r3) [inline grnet/website.dockerfile]: this too

File: grnet/website.dockerfile (hunk 1 above)
Comment (AEtherC0r3) [reply grnet/website.dockerfile]: https://github.com/nginxinc/docker-nginx/blob/master/stable/alpine/Dockerfile

File: grnet/mqtt.dockerfile [hunk 4]
```diff
@@ -3,21 +3,13 @@
 # Based on Debian
 ############################################################
 
-# Set the base image to Debian
-FROM debian:jessie
-
-# File Author / Maintainer
-MAINTAINER aldor
+# Set the base image to Debia
```
Comment (panagiks) [inline grnet/mqtt.dockerfile]: Debia != Debian :P

**Comment by panagiks** [comment]
Looks good to me (haven't tested it though)

**Pull request merged**